docs/content/node-operation/faq.mdx

```diff
@@ -50,15 +50,16 @@ They then join a webinar to meet the team and share more about how they’ll gro
 ### Is there any way to know if a node is currently online?
 
 To verify if a node is online, please [setup metrics](/node-operation/faq/#how-do-i-check-on-the-status-of-my-node) for the node.
-We also have an **internal** dashboard which is updated by pinging each of the registered node.
+We also have an **internal** dashboard which is updated by pinging each of the registered nodes.
```

Comment:
```suggestion
```

I'm not sure if it's worth it to mention an internal dashboard. It could be confusing for community members.